src/fastq/call_consensus_reads.rs

```diff
@@ -305,7 +305,7 @@ pub fn call_consensus_reads_from_paths(
         (false, false, true, true) => match CallNonOverlappingConsensusRead::new(
             &mut fastq::Reader::from_file(fq1).context(FastqIOError{ci_parameter: String::from("fq1"), path: String::from(fq1)})?,
             &mut fastq::Reader::from_file(fq2).context(FastqIOError{ci_parameter: String::from("fq2"), path: String::from(fq2)})?,
-            &mut fastq::Writer::new(GzEncoder::new(fs::File::create(fq1_out).context(FastqIOError{ci_parameter: String::from("fq1_out"), path: String::from(fq2_out)})?, Compression::default())),
+            &mut fastq::Writer::new(GzEncoder::new(fs::File::create(fq1_out).context(FastqIOError{ci_parameter: String::from("fq1_out"), path: String::from(fq1_out)})?, Compression::default())),
```

Comment:
A general remark: I think it is considered more idiomatic to write `fq1_out.to_owned()` instead of `String::from`.

Reply:
Good point. I revised the corresponding lines and did a bit of cargo-fmt magic, too.